src/main/resources/embedding-providers-config.yaml

```diff
@@ -1,13 +1,12 @@
-# custom properties for enabling vectorize method
-stargate:
-  jsonapi:
+# Settings for the embedding providers
+datastax:
```

Comment:
the config applies to open source as well right? seems like it should still be `stargate`